README.md

```diff
@@ -31,6 +31,61 @@ Here is a brief list of the available widgets:
 * **edit_box**: an editable text box (multi-line edit_boxes are not yet fully supported).
 * **scroll_frame**: a frame that has scrollable content.
 
+# Getting started 
+first, clone the project
```

Comment:
Let's get into the nitpicks now. Can you please ensure that:
 - each sentence starts with a capital letter and ends with adequate punctuation (dots, colons, etc.)?
 - each section has its name starting with a capital letter?